autojump_speed_multiplier

Gamersxp6248 opened the pull request:
Added to MTA so that it can still correctly be limited. Requires suicide if changed mid game. The SuperJumpMultiplier determines what your speed is multiplied by every time you jump, it's set to 1 in MTA, but can now be changed from 0.1 to 5 outside of wanted.

Defaults to 1.5 because that was the setting before this was added

Gamersxp6248 committed:
autojump_speed_multiplier

Added to MTA so that it can still correctly be limited. Requires suicide if changed mid game.

Gamersxp6248 commented:
I wouldn't be surprised if this gets denied because as far as I know this would make much more sense going into the superjump addon itself, but I just don't have access to that github

jule14 commented on lua/mta_libs/wanted_constraints.lua:
```diff
@@ -92,7 +99,7 @@ local function release(ply)
 
     if ply.SetSuperJumpMultiplier then
-        ply:SetSuperJumpMultiplier(1.5)
+        ply:SetSuperJumpMultiplier(ply:GetInfoNum("autojump_speed_multiplier", 1.5))
```
remove everything except this, i'll change the rest where you can't (eventually)

Gamersxp6248 replied:
👍 removing the other additions then

Python1320 commented on lua/mta_libs/wanted_constraints.lua:
```diff
@@ -20,6 +20,13 @@ local ply_ents_to_remove = {
 }
+
+gameevent.Listen( "player_connect" )
+hook.Add("player_connect", "PlayerEntityLoaded", function( data )
```
What was wrong with PlayerInitialSpawn hook?

Gamersxp6248 replied:
I'm not sure, for some reason the coder (Visual Code Studio) i was using was saying it was wrong. I wanted to put it in anyways but i'm not sure what the true standard is for this community rather it's "It just needs to work" or "My editor better make a smiley face".

Gamersxp6248 committed:
Reversed unnecessary changes

Supposedly these lines are being put into the actual superjump repo where they belong.

jule14 commented:
i probably got everything necessary, complain if it's inconsistent
